### PR TITLE
Update chart annotations

### PR DESCRIPTION
**Pull request by nwmac**
Update the chart annotations so that all charts are up to date.

### CODE REVIEW

File: scripts/publish
[the remark this answers is not in the document]
Comment (nwmac) [reply scripts/publish:1]: This PR is based on top of that one, so should be merged first

File: index.yaml
[the remark this answers is not in the document]
Comment (nwmac) [reply index.yaml:1]: Yes - thanks, working on that

File: scripts/publish
[the remark this answers is not in the document]
Comment (nwmac) [reply scripts/publish:1]: I'll just close 51

File: index.yaml
[the remark this answers is not in the document]
Comment (nwmac) [reply index.yaml:1]: Thanks @jordojordo

**Review by aalves08 — APPROVED** [review]
Chart update works great. Adding this repo + branch to Rancher and installing a "rebuilt" extension also works as expected.

Updates to annotations also look good.

Overall, LGTM! Thank you @nwmac 🙏

**Comment by aalves08** [comment]
Next steps here are to migrate this to `dashboard/master` and also `dashboard/release-2.9` (the reusable workflows should be pointing here, since `master` is "broken")

**Comment by nwmac** [comment]
@aalves08 How do you mean master is broken?

**Comment by aalves08** [comment]
`master` is "broken" in the sense that it's already using node 20 and that generates quite a few errors for people that wanted to publish extensions. We've advised people to point their workflows to the branch `release-2.9`, like:
https://github.com/neuvector/manager-ext/blob/main/.github/workflows/build.yaml#L21

**Comment by aalves08** [comment]
For context, the issue tracking this PR is: https://github.com/rancher/dashboard/issues/11416, although once we get the changes to `master` we could possibly consider it "done", imo

**Review by jordojordo — APPROVED** [review]

**Pull request merged**